modules/shadertools/src/modules/lighting/pbr-material/pbr-material.ts

```diff
@@ -149,17 +149,5 @@ export const pbrMaterial = {
     // #ifdef PBR_DEBUG
     scaleDiffBaseMR: 'vec4<f32>',
     scaleFGDSpec: 'vec4<f32>'
-  },
-
-  bindings: {
-    baseColorSampler: {type: 'texture', location: 8}, // #ifdef HAS_BASECOLORMAP
```

Comment:
Don't we need the locations?

Reply:
As far as I can tell `ShaderInputs` & `Model` don't expect a location, just a `bindings` map `Record<string, Binding>`. I changed `ShaderModule.bindings` to be used just for type inference to match `ShaderModule.uniforms` and this certainly works for WebGL. Do we need the explicit locations for WebGPU?

Reply:
We do need them but I think they are parsed from the WGSL, so removing them here should be OK.